web/src/components/speaker/Languages.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <v-col :class="dynamicPadding">
+    <v-row>
+      <div
+        v-for="language in languages"
+        :key="language.id"
+        class="d-flex justify-center align-center mr-2 mb-1"
+      >
+        <v-chip
+          color="silver"
+          outlined
+          small
+        >
+          {{ format(language) }}
+        </v-chip>
+      </div>
+    </v-row>
+  </v-col>
+</template>
+
+<script>
+import formatLanguage from '@/util/format';
+
+export default {
+  props: {
+    languages: {
+      type: Array,
+      required: false,
+      default: () => [],
+    },
+  },
+  computed: {
+    dynamicPadding() {
+      if (this.$vuetify.breakpoint.mdAndUp) {
+        return {
+
+        };
```

Comment:
Do we need the brackets and empty line here? maybe an empty string and semicolon would suffice?

Reply:
It seemed like the dynamic classes wanted an empty object, though I could neaten up the spacing. May I do that in a later PR?